module/.gitignore

```diff
@@ -0,0 +1 @@
+/gravity
```

Comment:
nit: Not sure this is needed? Was a script making a gravity folder locally?

Reply:
I develop in that directory, the binary gets built there.